src/vos/vos_internal.h

```diff
@@ -145,7 +145,7 @@ enum {
 extern uint32_t vos_agg_gap;
 
 #define VOS_AGG_GAP_MIN		20 /* seconds */
-#define VOS_AGG_GAP_DEF		60
+#define VOS_AGG_GAP_DEF         25
```

Comment:
Can we change it back to original 20 seconds (or even smaller)? Even with 20 seconds (2.6), we've already hit false ENOSPACE in large I/O workload (for the vos pool with decent size).

I know that restart/resend could burden the engine further, but I think we can fix that by adjusting restart/resend logic, right?